MM2RandoLib/Utilities/MiscHacks.cs

```diff
@@ -447,17 +452,19 @@ public static void PreventETankUseAtFullLife(Patch p)
             int prgOffset = 0x30010 - 0x4000;
             // Inject new jump subroutine at 0D:9296 (should be 0x352A6).
             int jsrLocation = 0x9296 + prgOffset;
-            var jsrBytes = new byte[]
+
+            Byte[] jsrBytes = new Byte[]
             {
                 0x20, 0x77, 0xBF,   // JSR $BF77
             };
+
             for(int offset = 0; offset < jsrBytes.Length; ++offset)
             {
                 p.Add(jsrLocation + offset, jsrBytes[offset], "Prevent E-Tank Use at Full Life");
             }
 
             // Subroutine to decrement E-Tank Count. Skips decrement if Life == 28.
-            var eTankSubroutineBytes = new byte[]
+            Byte[] eTankSubroutineBytes = new Byte[]
```

Comment:
Similar question as before. `byte` vs. `Byte`?

Reply:
no functional difference (same as the other comments)